components/src/components/SimpleDateTime/Component.ts

```diff
@@ -1,62 +1,74 @@
 /* tslint:disable */
-import { Components } from 'formiojs';
+import _ from "lodash";
```

Comment:
This got reintroduced again somehow.

Reply:
Fixed all Linting errors in next commit.